akka-http-core/src/main/scala/akka/http/impl/engine/rendering/BodyPartRenderer.scala

```diff
@@ -130,7 +130,7 @@ private[http] object BodyPartRenderer {
     case x: RawHeader if (x is "content-type") || (x is "content-length") =>
       suppressionWarning(log, x, "illegal RawHeader")
 
-    case x => r ~~ x ~~ CrLf
+    case x => r ~~ x
```

Comment:
OK, so the `~~ CrLf` can be skipped here (and in similar places) because it uses the new `~~` that checks the header and then adds the newline itself.